### PR TITLE
Minor improvements to the README of hbck2

### PR DESCRIPTION
**Pull request by lfrancke**

### CODE REVIEW

File: hbase-hbck2/README.md [hunk 1]
```diff
@@ -22,17 +22,15 @@ _HBCK2_ is the repair tool for Apache HBase clusters.
...
-the repair tool that shipped with _hbase-1.x_ (A.K.A _hbck1_).  Use _HBCK2_ in place of
-_hbck1_ making repairs against hbase-2.x clusters. _hbck1_ should not be run against an
-hbase-2.x install. It may do damage. While _hbck1_ is still bundled inside hbase-2.x
--- to minimize surprise -- it is deprecated, to be removed in _hbase-3.x_. Its
-write-facility (`-fix`) has been removed. It can report on the state of an hbase-2.x
-cluster but its assessments will be inaccurate since it does not understand the internal
-workings of an hbase-2.x.
+HBCK2 is the successor to [hbck](https://hbase.apache.org/book.html#hbck.in.depth), the repair tool that shipped with _HBase 1.x_ (A.K.A _hbck1_).
+Use _HBCK2_ in place of _hbck1_ making repairs against hbase-2.x clusters.
+_hbck1_ should not be run against an HBase 2.x installation as it may to damage.
```
Comment (wchevreuil) [inline hbase-hbck2/README.md:30]: nit: "may do damage"

**Review by wchevreuil — APPROVED** [review]
LGTM, just a minor nit.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 43s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  markdownlint  |   0m  0s |  markdownlint was not available.  |
| +0 :ok: |  shelldocs  |   0m  0s |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 30s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  spotless  |   0m 10s |  branch has no errors when running spotless:check.  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  3s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  shellcheck  |   0m  0s |  There were no new shellcheck issues.  |
| -1 :x: |  whitespace  |   0m  0s |  The patch has 2 line(s) that end in whitespace. Use git apply --whitespace=fix <<patch_file>>. Refer https://git-scm.com/docs/git-apply  |
| -1 :x: |  spotless  |   0m  3s |  patch has 22 errors when running spotless:check, run spotless:apply to fix.  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 13s |  The patch does not generate ASF License warnings.  |
|  |   |   2m 59s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.43 ServerAPI=1.43 base: https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-138/1/artifact/yetus-precommit-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase-operator-tools/pull/138 |
| Optional Tests | dupname asflicense spotless markdownlint shellcheck shelldocs |
| uname | Linux 545ecf54b405 5.4.0-1103-aws #111~18.04.1-Ubuntu SMP Tue May 23 20:04:10 UTC 2023 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/jenkins/hbase-personality.sh |
| git revision | master / a4af0cc |
| whitespace | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-138/1/artifact/yetus-precommit-check/output/whitespace-eol.txt |
| spotless | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-138/1/artifact/yetus-precommit-check/output/patch-spotless.txt |
| Max. process+thread count | 34 (vs. ulimit of 5000) |
| modules | C: hbase-hbck2 . U: . |
| Console output | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-138/1/console |
| versions | git=2.30.2 maven=3.8.6 shellcheck=0.7.1 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

File: hbase-hbck2/README.md (hunk 1 above)
Comment (lfrancke) [reply hbase-hbck2/README.md:30]: Thank you, I totally missed this review!

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m  8s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  markdownlint  |   0m  0s |  markdownlint was not available.  |
| +0 :ok: |  shelldocs  |   0m  0s |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 25s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  spotless  |   0m  6s |  branch has no errors when running spotless:check.  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  2s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  shellcheck  |   0m  0s |  There were no new shellcheck issues.  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  spotless  |   0m  5s |  patch has no errors when running spotless:check.  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 11s |  The patch does not generate ASF License warnings.  |
|  |   |   2m 13s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.43 ServerAPI=1.43 base: https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-138/2/artifact/yetus-precommit-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase-operator-tools/pull/138 |
| Optional Tests | dupname asflicense spotless markdownlint shellcheck shelldocs |
| uname | Linux 4801bd0ae502 5.4.0-1103-aws #111~18.04.1-Ubuntu SMP Tue May 23 20:04:10 UTC 2023 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/jenkins/hbase-personality.sh |
| git revision | master / a4af0cc |
| Max. process+thread count | 34 (vs. ulimit of 5000) |
| modules | C: hbase-hbck2 . U: . |
| Console output | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-138/2/console |
| versions | git=2.30.2 maven=3.8.6 shellcheck=0.7.1 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 25s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  markdownlint  |   0m  0s |  markdownlint was not available.  |
| +0 :ok: |  shelldocs  |   0m  0s |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 20s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  spotless  |   0m  6s |  branch has no errors when running spotless:check.  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  3s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  shellcheck  |   0m  0s |  There were no new shellcheck issues.  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  spotless  |   0m  4s |  patch has no errors when running spotless:check.  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 11s |  The patch does not generate ASF License warnings.  |
|  |   |   1m 25s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.43 ServerAPI=1.43 base: https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-138/3/artifact/yetus-precommit-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase-operator-tools/pull/138 |
| Optional Tests | dupname asflicense spotless markdownlint shellcheck shelldocs |
| uname | Linux be7d59696d5e 5.4.0-1103-aws #111~18.04.1-Ubuntu SMP Tue May 23 20:04:10 UTC 2023 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/jenkins/hbase-personality.sh |
| git revision | master / a4af0cc |
| Max. process+thread count | 34 (vs. ulimit of 5000) |
| modules | C: hbase-hbck2 . U: . |
| Console output | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-138/3/console |
| versions | git=2.30.2 maven=3.8.6 shellcheck=0.7.1 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Review by wchevreuil — APPROVED** [review]

**Pull request merged**